docs/european-user-consent.mdx

```diff
@@ -132,6 +149,9 @@ if (storeAndAccessInformationOnDevice === false) {
 }
 ```
 
+**Note:** Don't try to use this functionality to enforce user consent on iOS,
```

Comment:
This seems really important (since iOS app review is so hit or miss...) but at the same time is vague. Is there a link to some upstream document or a key phrase people could do a search for that expands on this? As an implementer the first thing I would want to do is click on a link that said something like "Find more information for respectfully obtaining user consent here" or similar

I looked myself at the UMP for iOS site and saw no mention of anything? https://developers.google.com/admob/ump/ios/quick-start